Automattic/jetpack
# Blaze: only display UI to admins.

## Proposed changes:

* Let's avoid displaying the Blaze UI to non-admins. Only site owners should be able to promote content on their site.

### Other information:


## Jetpack product discussion

* peeHDf-zm-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a public site that's connected to WordPress.com, with 2 **connected** users. One of those users should be the admin, the other an editor.
* Go to Posts menu and move your mouse over published posts;
    * When logged in as the editor, you should not see any link.
    * When logged in as the admin, you should see the link.
